Makefile

```
@@ -132,11 +132,28 @@ lighthouse:
+# is: the embedded vector plot differs by ~1 ULP across CPUs. See
+# scripts/check_reports.py.
+#
+# Renders into a scratch directory and swaps it in only once the whole set is
+# written. Clearing the output first, which is how this used to work, means a
+# generator that dies halfway leaves the working tree stripped of the committed
+# examples and the maintainer reaching for `git checkout`. The clearing itself
+# has to stay, because the generator only overwrites and never deletes, so a
+# fiche that is no longer produced would survive as a stale orphan and slip
+# past the staleness check; it just belongs after a successful run rather than
+# before an attempted one. The trap covers the interrupted run the same way.
 reports:
```

checkmake (0.3.2)

[warning] Target body for "reports" exceeds allowed length of 5 lines (7).

(maxbodylength)